plugins/com.aptana.ui/src/com/aptana/ui/dialogs/MultipleInputMessageDialog.java

```diff
@@ -206,13 +206,29 @@ else if (element instanceof JsonNode)
 						return element.toString();
 					}
 				});
-				combo.setSelection(new StructuredSelection(values.get(0)));
+				JsonNode defaultValue = question.path(DEFAULT);
+				if (defaultValue != null && values instanceof ArrayNode)
+				{
+					ArrayNode arrayNode = (ArrayNode) values;
+					for (JsonNode node : arrayNode)
+					{
+						if (ObjectUtil.areEqual(node.get(VALUE).textValue(), defaultValue.textValue()))
+						{
+							combo.setSelection(new StructuredSelection(node));
+							break;
+						}
+					}
```

Comment:
Value is not a direct string it's a JsonNode.
[{"name":"Appcelerator Studio","value":"100000780"},{"name":"AppcStudioFree","value":"100001523”}]